netlify.toml

```diff
@@ -0,0 +1 @@
+X-Frame-Options = ALLOW
```

Comment:
```suggestion
[[headers]]
  for = "/*"
  [headers.values]
    X-Frame-Options = "ALLOW"
```

I checked their docs and I think this is a correct structure

Reply:
just tried this...pushed